lib/screens/routing_screen.dart

```diff
@@ -1,145 +1,215 @@
-import 'package:easel_flutter/easel_provider.dart';
```

Comment:
shouldn't this whole file just be deleted?

Reply:
Done